test/EventStore.Client.Tests.Common/Fixtures/EventStoreTestCluster.cs

```diff
@@ -26,8 +26,7 @@ public class EventStoreTestCluster(EventStoreFixtureOptions options) : TestCompo
 			["EVENTSTORE_DISCOVER_VIA_DNS"]             = "false",
 			["EVENTSTORE_ENABLE_EXTERNAL_TCP"]          = "false",
 			["EVENTSTORE_STREAM_EXISTENCE_FILTER_SIZE"] = "10000",
-			["EVENTSTORE_STREAM_INFO_CACHE_CAPACITY"]   = "10000",
-			["EVENTSTORE_ENABLE_ATOM_PUB_OVER_HTTP"]    = "true" // why true?
```

Comment:
similarly to elsewhere, this might be handy to keep for debugging

Reply:
it was moved to the `GlobalEnvironment` class and made the default true.